### DISCUSSION TITLE
LNDhub and sharing channels

### DISCUSSION THREAD

**Discussion by stefment** [btcpayserver/btcpayserver#2633] [General]
As far as i understand in order to use BTCPayserver every business must manage its own node. This is not efficient due to the amount of skill it takes and although it can probably be learned in a week or two, it still seems like a bad idea. But i could be wrong.

What if BTCpayserver would combine with LNDHub. Then you could plug into someone elses LN node. He could possibly be managing a node that several business was linked up to. This way you have 1 guy managing liquidity and LN, making sure of high uptime and so on, for several business, rather than each business having to this separately. Discuss :)

**Comment by pavlenex** [comment]
Duplicate of https://github.com/btcpayserver/btcpayserver/discussions/1047

**Comment by dennisreimann** [comment]
LNDHub has recently been added to the Docker deployment, see btcpayserver/btcpayserver-docker#472
